Marlin/MarlinSerial.h

```diff
@@ -149,12 +149,12 @@ class MarlinSerial { //: public Stream
     void println(void);
 };
 
-extern MarlinSerial MSerial;
+extern MarlinSerial MarlinCustomizedSerial;
```

Comment:
You are correct about instance. I wasn't thinking about it too much and just followed "MSerial". So, it, too, was improperly formatted.


Do you want to do the global change, or shall I? (It's trivial to do in TextMate)